eng/scripts/DevOps-Recordings-Download.ps1

```diff
@@ -0,0 +1,62 @@
+param ($pr, $noWait = $false)
+
+$repoRoot = Resolve-Path "$PSScriptRoot/../..";
+$artifactsPath = Join-Path $repoRoot "artifacts"
+
+$token = (az account get-access-token --resource=https://management.core.windows.net/ -o json | ConvertFrom-Json).accessToken
```

Comment:
If you're already depending on PowerShell, why also depend on azure CLI? Why not just the Az powershell module? That's why I did that for all the TestResources scripts.

Reply:
I didn't find a devops powershell module.